build/BuildSteps.cs

```diff
@@ -157,18 +155,17 @@ public static void FilterPowershellRuntimes()
                     var powerShellVersion = Path.GetFileName(powershellWorkerPath);
                     var powershellRuntimePath = Path.Combine(powershellWorkerPath, "runtimes");
 
-                    var allKnownPowershellRuntimes = Settings.ToolsRuntimeToPowershellRuntimes[powerShellVersion].Values.SelectMany(x => x).Distinct().ToList();
                     var allFoundPowershellRuntimes = Directory.GetDirectories(powershellRuntimePath).Select(Path.GetFileName).ToList();
+                    var powershellForCurrentRuntime = Settings.ToolsRuntimeToPowershellRuntimes[powerShellVersion][runtime];
```

Comment:
I'm having some troubles understanding the `powershellForCurrentRuntime` variable name. Shouldn't this be something like `powershellRuntimesForCurrentToolsRuntime`?

Reply:
Yes, it should. I will update it.